MaxText/checkpointing.py

```diff
@@ -141,6 +141,60 @@ def create_orbax_emergency_replicator_checkpoint_manager(
   return manager
 
 
+def replicator_error_handler(config: Any):
+  """Replicator error handler to handle errors in replicator service."""
+  if (
+      config.enable_emergency_checkpoint
+      and config.use_replicator_service
+      and config.local_checkpoint_directory
+  ):
+    local_dir = config.local_checkpoint_directory
+    replicator_errors_file = f"{local_dir}/replicator.errors"
+    replicator_failed_file = f"{local_dir}/replicator.failed"
+    is_fatal = False
+    if replicator_error_handler_helper(replicator_failed_file):
+      is_fatal = True
+    replicator_error_handler_helper(replicator_errors_file)
+
+    if is_fatal:
+      raise ValueError(
+          "Replicator fatal error found in replicator.failed file."
+      )
+
+
+def replicator_error_handler_helper(error_file: str):
```

Comment:
Nit: Can you rename this method to something more descriptive than `helper`? It will make it easier to tell what the method is responsible for (even though you have described it in the comment)